microprofile.jdt/com.redhat.microprofile.jdt.core/src/main/java/com/redhat/microprofile/jdt/internal/core/ProjectLabelRegistry.java

```diff
@@ -0,0 +1,96 @@
+/*******************************************************************************
+* Copyright (c) 2019 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package com.redhat.microprofile.jdt.internal.core;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.resources.ResourcesPlugin;
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.runtime.IConfigurationElement;
+import org.eclipse.core.runtime.IExtensionRegistry;
+import org.eclipse.core.runtime.Platform;
+import org.eclipse.jdt.core.IJavaProject;
+import org.eclipse.jdt.core.JavaCore;
+import org.eclipse.jdt.ls.core.internal.ProjectUtils;
+
+import com.redhat.microprofile.commons.ProjectLabelInfo;
+import com.redhat.microprofile.jdt.core.IProjectLabelProvider;
+import com.redhat.microprofile.jdt.core.ProjectLabelDefinition;
+import com.redhat.microprofile.jdt.core.MicroProfileCorePlugin;
+
+/**
+ *  Registry to hold the extension point
+ * "com.redhat.microprofile.jdt.core.projectTypes".
+ * 
+ */
+public class ProjectLabelRegistry {
+
+	private static final String CLASS_ATTR = "class";
+
+	private static final String ID = "projectTypes";
```

Comment:
Replace (and rename) to 

`private static final String EXTENSION_PROJECT_LABEL_PROVIDERS = "projectLabelProviders";`